test/requirements_version_extract.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+""" Scan for CVE's in cve-bin-tool's pip requirements (main tool + docs)"""
+
+import csv
+import os
+import pkg_resources
+
+CACHEDIR = os.path.join(os.path.expanduser("~"), ".cache", "cve-bin-tool")
+ROOTPATH = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+DOCPATH = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "doc"))
+
+REQUIREMENTS_CSV = os.path.join(ROOTPATH, "requirements_vendor_product.csv")
+DOC_REQUIREMENTS_CSV = os.path.join(DOCPATH, "requirements_vendor_product.csv")
+
+"""Helper function that returns product_vendor_version pair list from a product_vendor_csv requirements file. """
+
+
+def get_product_vendor_version(filename):
+    product_vendor_version = []
+    with open(filename, "r") as file:
+        reader = csv.reader(file)
+        next(reader)
+        for vendor, product in reader:
+            if product.endswith("_notindb"):
+                product = product.replace("_notindb", "")
```

Comment:
I don't really understand this `_notindb` thing but shouldn't you also remove it from `vendor` then? Or maybe not include in the csv at all? I don't know :)